src/Utils/utils.ts

```diff
@@ -240,3 +245,147 @@ export const formatCurrency = (price: number) =>
 export const isUserOnline = (user: { last_login: DateLike }) => {
   return dayjs().subtract(5, "minutes").isBefore(user.last_login);
 };
+
+export interface CountryData {
+  flag: string;
+  name: string;
+  code: string;
+}
+
+export const parsePhoneNumber = (phoneNumber: string, countryCode?: string) => {
+  const phoneCodes: Record<string, CountryData> = phoneCodesJson;
+  let parsedNumber = phoneNumber.replace(/[-+() ]/g, "");
+  if (countryCode && phoneCodes[countryCode]) {
+    parsedNumber = phoneCodes[countryCode].code + parsedNumber;
+  } else if (!phoneNumber.startsWith("+")) {
+    return undefined;
+  }
+  parsedNumber = "+" + parsedNumber;
+  return parsedNumber;
+};
+
+export const isValidPhoneNumber = (phoneNumber: string) => {
+  if (phoneNumber === null) return false;
+  phoneNumber = phoneNumber.replace(/[-+.() ]/g, "");
+  if (phoneNumber.startsWith("+91")) {
+    return phoneNumber.length === 13;
+  } else if (phoneNumber.startsWith("1800")) {
+    return phoneNumber.length === 10 || phoneNumber.length === 11;
+  } else {
+    return phoneNumber.length <= 15 && phoneNumber.length >= 12;
+  }
+};
```

Comment:
```suggestion
```

@print-Sathvik https://github.com/coronasafe/care_fe/pull/5999 this PR introduced a proper validators that is in sync with the backend phone number validation too. 

Refactor your code to use `PhoneNumberValidator` instead of this function.

Reply:
https://github.com/coronasafe/care_fe/pull/5999/files#diff-c636e9ceaf17e1786a2c45ee0c55b24c47893a291e51cc06fad0de0cf2fd46df